### PR TITLE
Fix fonts assets

### PR DESCRIPTION
**Pull request by johnny243**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "start": "webpack-dev-server --progress",
     "bundle": "webpack --mode production",
-    "build": "bundle install && npm install && npm run bundle",
+    "build": "bundle install && npm install && bundle exec rails assets:precompile && npm run bundle",
```
Comment (karolsojko) [inline package.json:12]: npm install should be npm ci

Can you also adjust the Dockerfile so it doesn’t duplicate on running these commands?

Comment (johnny243) [reply package.json]: Done :+1:

**Comment by baptiste-grob** [comment]
Would this change work with the webpack-dev-server as well?

**Comment by karolsojko** [comment]
> Would this change work with the webpack-dev-server as well?

I've checked out @johnny243 branch and did `npm ci && npm start` and everything is looking good. Webpack serves from dist so that should be ok.

**Pull request merged**